.github/workflows/run_linter_powershell.yaml

```diff
@@ -0,0 +1,34 @@
+# Author: James Garriss
+# Purpose: Run the linter against the PowerShell code
+
+name: Lint Check for PowerShell
+
+# Run whenever PowerShell files are checked in
+on:
+  push:
+    paths:
+      - '**.ps1'
+      - '**.psm1'
+      - '**.psd1'
+      - '**.pssc'
+      - '**.psrc'
+      - '**.ps1xml'
+      - '**.cdxml'
+    branches:
+      - 700-add-powershell-linter
```

Comment:
Remove branch - would not trigger other branches (i.e., main, flipper, ...)
You could add this workflow to push path to get a trigger if updating this file on a push

Reply:
This was for testing purposes.  Removed.